src/main/java/com/welcome/bot/services/UserService.java

```diff
@@ -0,0 +1,66 @@
+package com.welcome.bot.services;
+
+import java.util.List;
+
+import org.modelmapper.ModelMapper;
+import org.modelmapper.TypeToken;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageImpl;
+import org.springframework.data.domain.Pageable;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Service;
+import org.springframework.web.server.ResponseStatusException;
+
+import com.welcome.bot.domain.User;
+import com.welcome.bot.models.UserDTO;
+import com.welcome.bot.repository.UserRepository;
+
+@Service
+public class UserService{
+	@Autowired
+	UserRepository userRepository;
+
+	public User AddUser( User user) {
+		userRepository.save(user);
+		return user;
+	}
+
+	public Iterable<User> GetUsers(Pageable pageable) {
+		Page<User> usersPage = userRepository.findAll(pageable);
+		return usersPage;
+	}
+
+	public User GetUser(Long id) {
+		User user = userRepository.findById(id).orElseThrow(() -> new RuntimeException("No data!"));
+		return user;
+	}
+	@Autowired
+	ModelMapper modelMapper;
```

Comment:
Keep dependencies at top level of class.

Reply:
Resolved.